bolt-modules/out/spec/verbose_spec.rb

```diff
@@ -0,0 +1,194 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'bolt/executor'
```

Comment:
This file is a duplicate of `bolt-modules/out/spec/functions/out/verbose_spec.rb` - that file is in the right location, this one can be removed.